runelite-client/src/main/java/net/runelite/client/plugins/corp/CorpDamageOverlay.java

```diff
@@ -82,8 +83,11 @@ public Dimension render(Graphics2D graphics)
 		int totalDamage = corpPlugin.getTotalDamage();
 		int players = corpPlugin.getPlayers().size();
 
-		// estimate how much damage is required for kill based on number of players
-		int damageForKill = players != 0 ? totalDamage / players : 0;
+		// estimate the probability of your damage exceeding everyone else's
+		double mean = players != 0 ? totalDamage / players : 0;
+		double sd = Math.sqrt(totalDamage / (0.1 * players));
```

Comment:
doesn't the division by 0 concern also carry over to this line? & can you explain how you arrived to this formulation?

Reply:
It does indeed! Well spotted

Reply:
Added a comment with the derivation and prevented the division by 0